tests/functional/spec/radio.spec.js

```diff
@@ -1,148 +0,0 @@
-const helpers = require('../helpers');
-const RadioMandatoryPage = require('../pages/surveys/radio/radio-mandatory.page');
-const RadioNonMandatoryPage = require('../pages/surveys/radio/radio-non-mandatory.page');
-const SummaryPage = require('../pages/surveys/radio/radio-summary.page');
-
-describe('Radio button with "other" option', function() {
-
-  var radio_schema = 'test_radio.json';
```

Comment:
do we still need the `test_radio.json` schema?

Reply:
Currently used in test_form_helper and test_questionaire_form. Could just replace them and edit them to use the new schemas?

Reply:
what is that testing and should it have its own schema rather than creating a dependency on schemas

Reply:
Its testing the mandatory other fields etc. so I'll just change it to use the ones that I created in this PR, should pass then as well and can scrap the `test_radio.json`